.github/workflows/devcontainer.yml

```diff
@@ -18,26 +36,38 @@ jobs:
           docker buildx build . \
```

Comment:
```suggestion
          echo "Building a development image"
          docker buildx build . \
```
maybe we can add some annotations for run steps.

Reply:
Good point. I created an issue #501 to add proper names, maybe we can do it there?

Reply:
Got it.